truenas/webui
# NAS-100829 / 21.08 / Checkbox for restart services during unlock

Changes:

- Added checkboxes on the Unlock form to choose services to be restarted during unlock
- Fix untilDestroyed error on volumes-list
- Replace `translateService.get` with `translateService.instant` for volumes-list-table-config

How to test:
1. Go to the Storage page and create an encrypted pool
2. Click "three dots" menu and choose "Encryption Options"
3. Switch Encryption Type from "Key" to "Passphrase" and save it
4. Click "three dots" menu and choose "Lock"
5. Click "three dots" menu again and choose "Unlock" you'll be redirected to the "Unlock" page
6. Now you can see checkbox "Check All" and available services below

For further details check Jira ticket or ping me
